examples/front-proxy/service.py

```diff
@@ -12,5 +26,19 @@ def hello(service_number):
                                     socket.gethostname(),
                                     socket.gethostbyname(socket.gethostname())))
 
+@app.route('/trace/<service_number>')
+def trace(service_number):
+    headers = {}
+    # call service 2 from service 1
+    if int(os.environ['SERVICE_NAME']) == 1 :
+        for header in TRACE_HEADERS_TO_PROPAGATE:
+            if header in request.headers:
+                headers[header] = request.headers[header]
+        ret = requests.get("http://localhost:9000/trace/2", headers=headers)
```

Comment:
You mean service_number + 1 ? Because the entire code is specialized to service version == 1